pkg/envtest/komega/default.go

```diff
@@ -0,0 +1,87 @@
+package komega
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// defaultK is the Komega used by the package global functions.
+var defaultK = &komega{ctx: context.Background()}
```

Comment:
No option to set the context on the default komega?

Reply:
Not sure how useful that is, but I've added it for completeness.